### PR TITLE
chore(docs): Added release notes 2022.14

### LINKED ISSUE #8300: Chore: release notes 2022.14
Chore: release notes 2022.14

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #8300 

**Additional issues**:

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,96 @@ id: Release notes
...
+  - Fixed to prevent error being thrown when using invalid variant ([#8229](https://github.com/patternfly/patternfly-react/pull/8229))
+- **Banner:**
+  - Added support for status icons ([#8195](https://github.com/patternfly/patternfly-react/pull/8195))
+  - Updated tests to new React testing library standards ([#8160](https://github.com/patternfly/patternfly-react/pull/8160))
+- **Button:**
+  - Added inline link progress support to examples ([#8172](https://github.com/patternfly/patternfly-react/pull/8172))
+- **Clipboard copy:**
+  - Added OUIA props to clipboard copy ([#8193](https://github.com/patternfly/patternfly-react/pull/8193))
+- **Code editor:**
+  - Used `codeEditorControls` and cleaned up[#7931](https://github.com/patternfly/patternfly-react/pull/7931))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:31]: Missing space & parenthesis
```suggestion
  - Used `codeEditorControls` and cleaned up ([#7931](https://github.com/patternfly/patternfly-react/pull/7931))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,96 @@ id: Release notes
...
+- **Data list:**
+  - Updated to pass button props to toggle ([#8222](https://github.com/patternfly/patternfly-react/pull/8222))
+- **Context selector:**
+  - Updated isFlipEnabled to true by default ([#8215](https://github.com/patternfly/patternfly-react/pull/8215))
+- **Dropdown:**
+  - Updated isFlipEnabled to true by default ([#8215](https://github.com/patternfly/patternfly-react/pull/8215))
+  - Add support for forward ref and updated documentation for dropdown next. ([#8142](https://github.com/patternfly/patternfly-react/pull/8142))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:38]: Remove period
```suggestion
  - Add support for forward ref and updated documentation for dropdown next ([#8142](https://github.com/patternfly/patternfly-react/pull/8142))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -3,6 +3,96 @@ id: Release notes
...
+  - Enabled right aligned dropdown with isFlipEnabled ([#8224](https://github.com/patternfly/patternfly-react/pull/8224))
+  - Added OUIA support to dropdown next ([#8135](https://github.com/patternfly/patternfly-react/pull/8135))
+- **Dual list selector:**
+  - Fixed Add/remove button that didn't work if search is used ([#8269](https://github.com/patternfly/patternfly-react/pull/8269))
+- **Empty state:**
+  - Added `EmptyStatePrimary` to documentation ([#8161](https://github.com/patternfly/patternfly-react/pull/8161))
+- **Label:**
+ Update aria-labels to include label text ([#8192](https://github.com/patternfly/patternfly-react/pull/8192))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:46]: Missing line break & bullet point, fix tense
```suggestion
  - Updated aria-labels to include label text ([#8192](https://github.com/patternfly/patternfly-react/pull/8192))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -3,6 +3,96 @@ id: Release notes
...
+  - Converted examples to TypeScript ([#8176](https://github.com/patternfly/patternfly-react/pull/8176))
+- **Skip to content:**
+  - Converted examples to TypeScript ([#8138](https://github.com/patternfly/patternfly-react/pull/8138))
+- **Slider:**
+  - Corrected tab order when input is above thumb ([#8190](https://github.com/patternfly/patternfly-react/pull/8190))
+  - Reverted tab order update ([#8273](https://github.com/patternfly/patternfly-react/pull/8273))
+- **Text area:**
+  - Revamped unit tests ([#8150](https://github.com/patternfly/patternfly-react/pull/8150))
+- **Text input:**
+   Converted examples to TypeScript ([#8137](https://github.com/patternfly/patternfly-react/pull/8137))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:72]: Line break/bullet point missing
```suggestion
  - Converted examples to TypeScript ([#8137](https://github.com/patternfly/patternfly-react/pull/8137))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 5]
```diff
@@ -3,6 +3,96 @@ id: Release notes
...
+  - Added OUIA props to `WizardNav` and `WizardNavItem` ([#8193](https://github.com/patternfly/patternfly-react/pull/8193))
+  - Updated wizard next to allow for `WizardStep` to better control state from props entry-point and included index ([#8218](https://github.com/patternfly/patternfly-react/pull/8218))
+
+### Topology
+- Added option to fit layout to screen upon layout completion ([#8210](https://github.com/patternfly/patternfly-react/pull/8210))
+- Allowed tooltip usage on pipeline task node badges ([#8208](https://github.com/patternfly/patternfly-react/pull/8208))
+- Added pipeline groups ([#8278](https://github.com/patternfly/patternfly-react/pull/8278))
+
+### Other
+-  **Chore**
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:87]: Extra space looks to be breaking indentation of 3 following lines
```suggestion
- **Chore**
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
Typo nits

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**